### PR TITLE
Adds current player and switch player functions

### PR DESCRIPTION
**Pull request by katemalone**
I'm 90% sure that will work but am not positive without the testing. Please Review. @slkarsh @Garrett-Iannuzzi @ec-myers

### CODE REVIEW

File: src/Round.js [hunk 1]
```diff
@@ -4,23 +4,41 @@ import Puzzle from './Puzzle.js';
 import Wheel from './Wheel.js';
 
 class Round {
-  constructor(player, puzzle, game, wheel) {
+  constructor(players, playerOne, playerTwo, playerThree, currentPlayer, puzzle, game, wheel) {
```
Comment (ec-myers) [inline src/Round.js]: We shouldn't need to pass the players individually through the constructor as we will have access to the array of players, as well as the currentPlayer.

File: src/Round.js [hunk 2]
```diff
@@ -4,23 +4,41 @@ import Puzzle from './Puzzle.js';
...
-  
+    this.wheel = wheel;
+    this.player1 = new Player(playerOne);
+    this.player2 = new Player(playerTwo);
+    this.player3 = new Player(playerThree);
+    this.currentPlayer = currentPlayer; 
+    // this.currentTurn = this.startTurn();
+  }
 
+  getStartingPlayer() {
```
Comment (ec-myers) [inline src/Round.js]: At the start of the game we can assign the first player to always be ```newGame.players[0]```, so we don't necessarily need to find the first player. Then when we need to find the next player during the game we will invoke the switchPlayer method below, which looks great. Let me know what you think about deleting the getStartingPlayer method.

File: src/Round.js [hunk 3]
```diff
@@ -4,23 +4,41 @@ import Puzzle from './Puzzle.js';
...
+    if(game.roundCounter % 2 === 0) {
+      return this.player1
+    } else {
+      return this.swithchPlayer()
+    }
   }
 
-  getCurrentPlayer() {
-    // this.player = 
+  swithchPlayer() {
```
Comment (ec-myers) [inline src/Round.js]: Typo in method name

File: src/Round.js (hunk 2 above)
Comment (slkarsh) [reply src/Round.js]: Agree about not needing the getStartingPlayer method, unless we want to use that for rounds after round 1, in which case we can add in conditional logic about who has the most/least money.

**Pull request merged**